guardian/support-frontend
# One time checkout - get user type

## What are you doing in this PR?

Returning the identity user type for one time payments through the API, and using this information on the client to set a URL param on the thank you page.

This client side solution does not work for PayPal, as we redirect them to the site and can't set the URL params



## How to test

Use the new one time checkout and see "userType=x" when redirected to the thank you page. 

I used the [always authenticate](https://docs.stripe.com/testing?locale=en-GB#regulatory-cards) card to simulate 3DS, as that makes an additional API call 


## Screenshots
![image](https://github.com/user-attachments/assets/66f61e6e-2a5d-4952-8b6a-e8a3f95d0fc7)
